### DISCUSSION TITLE
Update field in another thread

### DISCUSSION THREAD

**Discussion by aimgdev** [hydrostack/hydro#12] [Q&A] [answered]
How would I use another thread to update a field and the UI?
For example:
```Count: <strong>@Model.Count</strong>```

    public int Count { get; set; }
    public void IncTest()
    {
        Count++;
        ThreadPool.QueueUserWorkItem(delegate
        {
            Thread.Sleep(2000);
            Count++;
        });

    }
    
(In Blazor one would use StateHasChanged)

**Comment by kjeske** [comment, marked as answer]
You could use polling here ([docs](https://usehydro.dev/features/long-polling.html)):

```c#
public class MyComponent : HydroComponent
{
    public int Count { get; set; }

    [Poll(Interval = 2000)]
    public void IncTest()
    {
        Count++;
    }
}
```
